src/dropWhile.js

```diff
@@ -1,6 +1,7 @@
 var _curry2 = require('./internal/_curry2');
 var _dispatchable = require('./internal/_dispatchable');
 var _xdropWhile = require('./internal/_xdropWhile');
+var slice = require('./slice');
```

Comment:
i believe the internal references were deliberately removed to simplify the dependency tree?

Reply:
A big and long eclipse that hide something 😁, thanks for merging.